benchmark/buffers/buffer-copy.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+const common = require('../common.js');
+
+const bench = common.createBenchmark(main, {
+  size: [10, 1024, 2048, 4096, 8192],
+  n: [1024]
+});
+
+function main({ n, size }) {
+  const source = Buffer.allocUnsafe(size);
```

Comment:
This basically copies from an unitialized memory chunk.
Perhaps use a single source, initialized with crypto or fill?